contrib/packaging/docker/Dockerfile.runtime

```diff
@@ -61,6 +49,17 @@ make -j `getconf _NPROCESSORS_ONLN` && \
 strip bpftool && \
 cd ../../../../ && \
 #
+# clang/llvm image with only BPF backend
+#
+git clone --depth 1 -b master https://github.com/llvm/llvm-project.git llvm && \
```

Comment:
@borkmann Did you get a chance to test this on some of the builds affected by #10517 yet? I was under the impression newer versions of Clang actually made things worse. Alexei's commit message reports higher numbers
of states processed for newer Clang versions.

Reply:
> @borkmann Did you get a chance to test this on some of the builds affected by #10517 yet? I was under the impression newer versions of Clang actually made things worse. Alexei's commit message reports higher numbers of states processed for newer Clang versions.

I just did a comparison printing all complexity numbers to the kernel log from daemon startup and creation of a single endpoint. While clang 7 seems to do better in our init.sh (progs to the left), it seems to generate problematic code for bpf_lxc (progs to the right), and for latest clang-11 the situation is a bit the other way round.

![clang 7 vs clang 11 on bpf-next tree](https://user-images.githubusercontent.com/677393/76430600-c8a6f400-63b0-11ea-9989-261910f1e70f.png)

Reply:
Switching to mcpu=v2 overall improves it. New chart with clang 11:

![clang 7 vs clang 11 on bpf-next tree, mcpu=probe_v2](https://user-images.githubusercontent.com/677393/76440789-aae08b80-63be-11ea-863f-37ab12106ad9.png)